.github/workflows/continuous-integration.yml

```diff
@@ -1,6 +1,10 @@
 name: Check build pass/fail
 
-on: [pull_request, push]
+on:
+  pull_request:
```

Comment:
Can you make this account for pull requests across all branches, ie. a branch merging into another branch that's not master?

Reply:
This does do that.